public/index.ejs

```diff
@@ -244,7 +260,15 @@ async function populateFollowers() {
 
   while (nextLink) {
     console.log(`Fetching nextLink=${nextLink}`);
-    let response = await fetch(nextLink, {headers: kGitHubHeaders});
+    // create a new AbortController and add it the list
+    const controller = new AbortController();
+    fetchAbortHandler.push({nextLink: controller})
```

Comment:
You're pushing to the function here which is wrong.

Reply:
Ah, this was a dumb typo..